plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -248,17 +274,49 @@ func actionHandler(request *service.ActionRequest) error {
 }
 
 // traceflowHandler handlers the layout of Traceflow page.
-func traceflowHandler(request service.Request) (component.ContentResponse, error) {
+func (a *antreaOctantPlugin) traceflowHandler(request service.Request) (component.ContentResponse, error) {
 	layout := flexlayout.New()
 	card := component.NewCard(component.TitleFromString(antreaTraceflowTitle))
+
+	// Construct the available values of destination types.
+	dstTypeSelect := make([]component.InputChoice, len(opsv1alpha1.SupportedDestinationTypes))
+	for i, t := range opsv1alpha1.SupportedDestinationTypes {
+		dstTypeSelect[i] = component.InputChoice{
+			Label:   t,
+			Value:   t,
+			Checked: false,
+		}
+		// Set the default destination type.
+		if t == opsv1alpha1.DstTypePod {
+			dstTypeSelect[i].Checked = true
+		}
+	}
+
+	// Construct the available values of protocols.
+	protocolSelect := make([]component.InputChoice, len(opsv1alpha1.SupportedProtocols))
+	i := 0
+	for p, _ := range opsv1alpha1.SupportedProtocols {
+		protocolSelect[i] = component.InputChoice{
+			Label:   p,
+			Value:   p,
+			Checked: false,
+		}
+		// Set the default protocol.
+		if p == "TCP" {
```

Comment:
So we now use TCP as default protocol rather than ICMP previously?
The octant-plugin part of change LGTM, but I think we need to update https://github.com/vmware-tanzu/antrea/blob/master/docs/traceflow-guide.md regarding the changes.

Reply:
Currently, users need to choose from the three protocols supported and ICMP is the default protocol shown in UI. However, if we do not set the protocol when creating CRD, the protocol will still be ICMP in default.

Reply:
![image](https://user-images.githubusercontent.com/32829088/89769000-03af7e00-db2f-11ea-9ddf-4ee50989759f.png)
An example image of how UI sets the default protocol as TCP look like